README.md

```diff
@@ -2,11 +2,30 @@
 
 [OpenSidewalkMap](https://www.opensidewalkmap.com/)
 
-Based on [OpenParkingMap](https://github.com/brandonfcohen1/openparkingmap)
-
 Use data from [OpenStreetMap](https://www.openstreetmap.org/) to highlight sidewalks on the map. Lets you easily see what places are accesible by foot and what sidewalks are missing
+## Acknowledgements
+
+ - This project is based on a fork of [OpenParkingMap](https://github.com/brandonfcohen1/openparkingmap)
+ - Not to be confused with the OSWM (also "OpenSidewalkMap") project by [Kauê Vestena](https://github.com/kauevestena). You can access the main repository of that project [here](https://github.com/kauevestena/opensidewalkmap)
+## Installation
 
-### About the Name
-Not to be confused with the OSWM (also "OpenSidewalkMap") project by [Kauê Vestena](https://github.com/kauevestena).
+To install this project, you will require an API key from mapbox. Go to https://www.mapbox.com/, create an account and API key before continuing.
 
-You can access the main repository of that project [here](https://github.com/kauevestena/opensidewalkmap) 
+1. Clone the project
+    ```bash
+    git clone https://github.com/CurtisGreen/opensidewalkmap.git
+    ```
+2. Install dependencies
+    ```bash
+    npm install
+    ```
+3. Configure environment variables
+    ```bash
+    export NEXT_PUBLIC_MAPBOX_TOKEN=TOKEN_HERE
+    ```
```

Comment:
Should put something about using this or the `.env` file

Reply:
And maybe link to mapbox site

Reply:
I can do this. I was fiddling with using .env.local last evening, trying to avoid storing secrets in plain text using 1Password. I had an issue with getting the secrets to de-reference. I am aware that `.env.local` will be ignored in commits and the plain text method does work. I will write something to this effect.

Reply:
I've always just used `.env`, so I'm not familiar with `.env.local` but whatever works

Reply:
These instructions are now updated to communicate multiple available methods for parameterizing environment variables.